web/controllers/user_controller.ex

```diff
@@ -0,0 +1,29 @@
+defmodule UeberauthProvider.UserController do
+  use UeberauthProvider.Web, :controller
+
+  alias UeberauthProvider.User
+
+  def index(conn, params) do
+    users = Repo.all(User)
+    render conn, "index.html", users: users
+  end
+
+  def new(conn, params) do
```

Comment:
No, because in the `create` action you are just `render`ing the `new` action if there is an error. It assigns the error `changeset` so that it will re-render the form using the error changeset.